docs/config.json

```diff
@@ -756,14 +768,20 @@
     },
     {
       "timeline": [
+        {
+          "moved": 1,
+          "ringId": "adopt",
+          "date": "2023-06-05",
+          "description": "Works out-of-the-box with Ruby 3. Actively supported, unlike Synchrony (which is supplants)."
```

Comment:
⛏️ Noticed you were fixing typos - when lurking in this PR I found a small one
~(which is supplants)~ => (which it supplants)

Reply:
Thanks for catching that. Fixed: https://github.com/Invoca/tech-radar/pull/7/commits/4a9567eab97c390d1e09b2d4f5313611c612d618